static/src/javascripts/projects/commercial/modules/dfp/on-slot-load.ts

```diff
@@ -22,11 +22,13 @@ export const onSlotLoad = (event: googletag.events.SlotOnloadEvent): void => {
 		console.log('No iFrame found for slot', advert.id, advert.slot);
 		return;
 	}
-	postMessage(
-		{
-			id: iframe.id,
-			host,
-		},
-		iframe.contentWindow,
-	);
+	if (iframe.contentWindow) {
```

Comment:
We seem to use the early return pattern above, so I would use the same strategy here:

```ts
if(!iframe.contentWindow) {
    console.log('Some message', iframe, advert.id, advert.slot);
    return;
}
```